browser/brave_vpn/dns/brave_vpn_dns_observer_service_win.cc

```diff
@@ -32,6 +35,10 @@ namespace {
 const char kCloudflareDnsProviderURL[] =
     "https://chrome.cloudflare-dns.com/dns-query";
 
+// Timer to recheck the service launch after some time and fallback to DoH if
+// the service was not launched.
+const int kHelperServiceStartTimeoutSec = 3;
```

Comment:
Let's use `constexpr` here and above also.

Reply:
done